src/styles/blocks/main.scss

```diff
@@ -0,0 +1,18 @@
+@mixin size {
+  width: 800px;
+  height: 800px;
+}
```

Comment:
Better to pass variable with size here, then we can reuse this mixin for any square block